README.md

```diff
@@ -1,39 +1,58 @@
-# Detecting generic gene expression signals
+# Generic transcriptional responses revealed using _SOPHIE_: Specific cOntext Pattern Highlighting In Expression data
 
-**Alexandra J. Lee, Rani K. Powers, Dallas L. Mould, Dongbo Hu, Georgia Doing, Jake Crawford, James C. Costello, Deborah A. Hogan, Casey S. Greene**
+**Alexandra J. Lee, Dallas L. Mould, Jake Crawford, Dongbo Hu, Rani K. Powers, Georgia Doing, James C. Costello, Deborah A. Hogan, Casey S. Greene**
 
 **University of Pennsylvania, University of Colorado Anschutz Medical Campus, Dartmouth College**
 
-**Rationale**: People performing differential expression (DE) analysis found that some genes and subsequent pathways are more likely to be differentially expressed even across a wide range of experimental designs ([Powers et. al., Bioinformatics 2018](https://academic.oup.com/bioinformatics/article/34/13/i555/5045793 ); [Crow et. al., PNAS 2019](https://www.pnas.org/content/116/13/6491)). 
+<!--- [![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://academic.oup.com/gigascience/article/9/11/giaa117/5952607) --->
```

Comment:
This looks like a link to your previous GigaScience paper - is this what you want here, or is this a placeholder for when you post a preprint of this work?

Reply:
yes this is just a placeholder for now to remind me to add this information